content/guides/core-concepts/cypress-app.md

```diff
@@ -159,6 +159,22 @@ The title of each run is taken from the
 for that change, and clicking on it will take you to the corresponding run page
 in Cypress Cloud.
 
+### The Debug Page
+
+<DocsImage src="/img/guides/core-concepts/cypress-app/debug-page.png" alt="The Debug Page"></DocsImage>
+
+Like the Project Runs page above, the Debug Page surfaces data from Cypress
+Cloud (see [note above](#Overview)) directly inside Cypress App. However, this
```

Comment:
Per the style guide (https://docs.google.com/document/d/1pOZ0Ntm4b1Eev_iY5vLBPhOxx66Z8Njrvvg6WMCvWRI/edit), this "App" shouldn't be capitalized: the Cypress app.